apps/test/unit/templates/sectionProgress/VirtualizedDetailViewTest.js

```diff
@@ -27,15 +27,27 @@ describe('VirtualizedSummaryView', () => {
     stubRedux();
     registerReducers({sectionProgress, scriptSelection, currentUser});
     defaultProps = {
-      levelsByLesson: {
+      levelProgressByStudent: {
```

Comment:
Should be able to use `fakeStudentLevelProgress` from progressTestHelpers, once it's merged in

Reply:
these tests are going away once we finish merging our refactor! :)